specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -623,8 +624,22 @@
           }
         },
         "storageName": {
-          "description": "Name of storage resource. No need to provide for EmptyDir.",
+          "description": "Name of storage resource. No need to provide for EmptyDir and Secret.",
           "type": "string"
+        },
+        "secrets": {
+          "description": "Secrets to be added in volume. If no secrets are provided, all secrets in collection will be added to volume.",
+          "type": "object",
```

Comment:
Should this be `array` and items are the object defined here?

Reply:
yes! fixed it :)